### ISSUE TITLE
errors with nvidia-container-toolkit and nvidia-docker2

### ISSUE THREAD

**Issue by zeke** [replicate/codespaces#1]
Hey @andreasjansson and @chenxwh 👋🏼 

I ran this in my Codespaces env:

```
curl https://storage.googleapis.com/replicate-public/codespaces/install-cog.sh | bash
```

And got this at the end of the output:

```
Errors were encountered while processing:
 nvidia-container-toolkit
 nvidia-docker2
```

Is that something I should be concerned about?

Full output: https://gist.github.com/zeke/8661da0fab3defd13911180fb807ff93

**Comment by chenxwh** [comment]
I don't think I saw this error before, but tried on a new codespace instance and experienced the same. 
See error stating [here](https://gist.github.com/zeke/8661da0fab3defd13911180fb807ff93#file-gistfile1-txt-L90), so I added `nvidia-container-toolkit-base` to `install-cog.sh` and works fine for me. 

see https://github.com/replicate/codespaces/pull/2

**Comment by zeke** [comment]
Resolved by #2. Thanks!
